### PR TITLE
make kafka wait for zookeeper

### PR DESCRIPTION
**Pull request by snopoke**

### CODE REVIEW

File: docker/hq-compose.yml [hunk 1]
```diff
@@ -88,12 +88,13 @@ services:
...
 
   kafka:
-      image: wurstmeister/kafka:0.8.2.2
+      image: wurstmeister/kafka:0.8.2.2-1
       ports:
         - "9092:9092"
       environment:
           KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
           KAFKA_BROKER_ID: 1
+          CUSTOM_INIT_SCRIPT: "curl https://raw.githubusercontent.com/vishnubob/wait-for-it/master/wait-for-it.sh > wait.sh && chmod +x ./wait.sh && ./wait.sh zookeeper:2181 || exit 1"
```
Comment (dannyroberts) [inline docker/hq-compose.yml]: I think if we're going to use a github link it should be a permanent link (like https://github.com/vishnubob/wait-for-it/blob/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh) and not one that references an ever-changing branch (master).

**Review by dannyroberts — CHANGES_REQUESTED** [review]
Thanks for debugging, this looks great pending the one suggestion

File: docker/hq-compose.yml (hunk 1 above)
Comment (snopoke) [reply docker/hq-compose.yml]:
```suggestion
          CUSTOM_INIT_SCRIPT: "curl https://github.com/vishnubob/wait-for-it/blob/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh > wait.sh && chmod +x ./wait.sh && ./wait.sh zookeeper:2181 || exit 1"
```

Comment (snopoke) [reply docker/hq-compose.yml]: updated

File: docker/hq-compose.yml [hunk 2]
```diff
@@ -88,12 +88,13 @@ services:
...
 
   kafka:
-      image: wurstmeister/kafka:0.8.2.2
+      image: wurstmeister/kafka:0.8.2.2-1
       ports:
         - "9092:9092"
       environment:
           KAFKA_ZOOKEEPER_CONNECT: zookeeper:2181
           KAFKA_BROKER_ID: 1
+          CUSTOM_INIT_SCRIPT: "curl https://github.com/vishnubob/wait-for-it/blob/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh > wait.sh && chmod +x ./wait.sh && ./wait.sh zookeeper:2181 || exit 1"
```
Comment (snopoke) [inline docker/hq-compose.yml]:
```suggestion
          CUSTOM_INIT_SCRIPT: "curl https://raw.githubusercontent.com/vishnubob/wait-for-it/c096cface5fbd9f2d6b037391dfecae6fde1362e/wait-for-it.sh > wait.sh && chmod +x ./wait.sh && ./wait.sh zookeeper:2181 || exit 1"
```

**Review by dannyroberts — APPROVED** [review]

**Comment by dannyroberts** [comment]
Oops sorry for the wrong link.

**Pull request merged**